modules/tinymce/CHANGELOG.md

```diff
@@ -8,113 +8,134 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - New `editor.options` API to replace the old `editor.settings` and `editor.getParam` APIs #TINY-8206
-- The `tabindex` attribute is now copied from the target element to the iframe #TINY-8315
-- New `AutocompleterStart`, `AutocompleterUpdate` and `AutocompleterEnd` events #TINY-8279
-- New `mceAutocompleterClose`, `mceAutocompleterReload` commands #TINY-8279
-- New `FakeClipboard` API on the `tinymce` global #TINY-8353
-- New `slider` dialog component #TINY-8304
-- New `buttonType` property on dialog button components, supporting `toolbar` style in addition to `primary` and `secondary` #TINY-8304
-- New `imagepreview` dialog component, allowing preview and zoom of any image URL #TINY-8333
 - New `editor.annotator.removeAll` API to remove all annotations by name #TINY-8195
 - New `Resource.unload` API to make it possible to unload resources #TINY-8431
+- New `FakeClipboard` API on the `tinymce` global #TINY-8353
 - New `dispatch()` function to replace the now deprecated `fire()` function in various APIs #TINY-8102
```

Comment:
Should this be "change" instead of New? dispatch() replaces fire()

Reply:
We didn't remove the `fire` function so the `dispatch` function is actually new.

Reply:
Hmm, true. I think it was basically a rename? Not entirely sure

Reply:
It sort of was but largely wasn't, as the change added a new function and then replaced all internal uses to use the new name, instead of the deprecated one. Then as a last step it deprecated the old function name, which is why there's an entry in Deprecated for that as well.

Reply:
Right, makes sense. We should probably keep it as it is then